src/app/pages/login/login-page/login.component.ts

```diff
@@ -1,8 +1,9 @@
+import { TokenStorageService } from 'src/app/services/token-storage.service';
 import { Component, OnInit } from '@angular/core';
 import { FormControl, Validators } from '@angular/forms';
 import { MatSnackBar } from '@angular/material/snack-bar';
+import { ActivatedRoute } from '@angular/router';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SimpleSurveyProject/SimpleSurvey-Frontend/pullRequest?prid=7430210)
```suggestion
import { ActivatedRoute } from "@angular/router";
```